examples/jsm/interactive/HTMLMesh.js

```diff
@@ -461,7 +461,7 @@ function html2canvas( element ) {
 
 				} else {
 					
-					element.addEventListener( 'load', drawImage( element, x, y, width, height ) );
+					element.addEventListener( 'load', () => drawImage( element, x, y, width, height ) );
```

Comment:
I'm afraid that now callback removal would not work, because `drawImage !== () => drawImage( args )`...

Reply:
You're right, I should have a fix for this as well

Reply:
Too bad that that PR cannot be merged because of not handling properly when image is not loaded yet. Maybe I'm wrong but I would have not even bother handling that case in the lib, because update `htmlMesh.material.map.update()` can be fired from user code checking when image are loaded or every 1/30s or 1/60s like done with the statgui example https://threejs.org/examples/?q=vr#webxr_vr_sandbox

Reply:
@LeviPesin I don't understand your explanation, for me when a dom element HTMLImageElement will be removed, the call back will be garbage collected automatically

Reply:
I meant that the line `element.removeElementListener( drawImage )` does not work.

Reply:
@LeviPesin you are right, I missed that line